starlette_admin/contrib/beanie/view.py

```diff
@@ -0,0 +1,343 @@
+                    sa.StringField,
+                    sa.TextAreaField,
+                    sa.EmailField,
+                    sa.URLField,
+                    sa.PhoneField,
+                    sa.ColorField,
+                ]
+            ):
+                queries.append(Q(field.name, term, "$icontains"))
+        if queries:
+            return (
+                functools.reduce(lambda q1, q2: QCombination(Q.OR, [q1, q2]), queries),
+                False,
+            )
+        return Q.empty(), False
+
+    async def count(
+        self, request: Request, where: Union[Dict[str, Any], str, None] = None
+    ) -> int:
+        query, _ = await self._build_query(request, where)
+        if isinstance(query, QCombination):
+            flattened_query = flatten_qcombination(query)
+            result = self.document.find(flattened_query)
+        else:
+            if not bool(query):
+                return (
+                    await self.document.get_motor_collection().estimated_document_count()
+                )
+            result = self.document.find(query.query)
+        return await result.count()
+
+    async def find_all(
+        self,
+        request: Request,
+        skip: int = 0,
+        limit: int = 100,
+        where: Union[Dict[str, Any], str, None] = None,
+        order_by: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> List[Dict]:
+        if not where:
+            where = {}
+        query, is_full_text_query = await self._build_query(request, where)
+        if isinstance(query, QCombination):
+            flattened_query = flatten_qcombination(query)
+            result = self.document.find(flattened_query, **kwargs)
+        else:
+            result = self.document.find(query.query, **kwargs)
+
+        # handle order_by
+        if is_full_text_query and self.full_text_override_order_by:
+            result = result.sort(("score", {"$meta": "textScore"}))  # type: ignore
+        elif order_by:
+            result = result.sort(build_order_clauses(order_by))
+        return await result.skip(skip).limit(limit).to_list()
+
+    async def find_by_pk(
+        self, request: Request, pk: PydanticObjectId
+    ) -> Optional[Document]:
+        if not isinstance(pk, PydanticObjectId):
+            try:
+                pk = PydanticObjectId(pk)
+            except bson.errors.InvalidId:
+                return None
+
+        return await self.document.get(pk)
+
+    async def find_by_pks(
+        self, request: Request, pks: Iterable[PydanticObjectId]
+    ) -> List[Document]:
+        docs = []
+        for pk in pks:
+            doc = await self.document.get(pk)
+            if doc:
+                docs.append(doc)
+        return docs
+
+    async def get_pk_value(self, request: Request, obj: Any) -> Any:
+        if isinstance(obj, Link):
+            return getattr(obj.ref, not_none(self.pk_attr))
+
+        return getattr(obj, not_none(self.pk_attr))
+
+    async def serialize(
```

Comment:
Was it necessary to override this?

Reply:
On second thought, I found a better solution instead of overriding serialize. Fixed in 
3c9850e